### DISCUSSION TITLE
Is it possible to get intellisense in vscode?

### DISCUSSION THREAD

**Discussion by ghostflyby** [dotnet/interactive#2901] [Q&A]
The Jupyter-kernel ipynbs have intellisense completions and syntax highlighting from the pylance extension

**Comment by jonsequitur** [comment]
If you're not seeing IntelliSense, that could be a bug. What language and kernel are you using?

**Reply by ghostflyby** [reply]
For PowerShell, there is intellisense but not that “intelli”, all the completions are marked as “words” without any type or documentation.
For javascript, there is no intellisense at all.
